nd4j/nd4j-tensorflow/src/main/java/org/nd4j/tensorflow/conversion/graphrunner/GraphRunner.java

```diff
@@ -276,6 +313,23 @@ public GraphRunner(byte[] graphToUse,List<String> inputNames,org.tensorflow.fram
     }
 
 
+    private void initOptionsIfNeeded() {
+        //setup the status object to be used for all tensorflow calls
+        if(status == null) {
+            status = TF_NewStatus();
+        }
+
+        if (options == null) {
+            options = TF_NewSessionOptions();
+            if(protoBufConfigProto != null) {
+                BytePointer bytePointer = new BytePointer(protoBufConfigProto.toByteArray());
+                TF_SetConfig(options,bytePointer,bytePointer.getStringBytes().length,status);
+                if (TF_GetCode(status) != TF_OK) {
+                    throw new RuntimeException("ERROR: Unable to set value configuration:" + TF_Message(status).getString());
```

Comment:
Same thing..IllegalStateException.

Reply:
It was already like that, I just moved it.